app/TipsDetails.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dimensions, ScrollView, StyleSheet, Text, View } from 'react-native';
+
+import { TIPS_INFO } from './data/TipsInfo';
+import LightbulbIcon from './icons/LightbulbIcon';
+
+const screenHeight = Dimensions.get('window').height;
+const screenWidth = Dimensions.get('window').width;
+
+const TipsHeader = ({ schoolName }: { schoolName: string }) => (
+  <View style={style.headerContainer}>
+    <Text style={style.header}>{schoolName}</Text>
+  </View>
+);
+
+const TipsDetails = ({ schoolName }: { schoolName: string }) => {
+  const tipArray = []; //create new array to keep track of which tips are from the school
+
+  console.log(schoolName);
+
+  // loops over TIPS_INFO object and pulls which key:value pairs coordinate to the relevant school
+  for (const key in TIPS_INFO) {
+    const value = TIPS_INFO[key];
+    if (value.site === schoolName) {
+      tipArray.push(value);
+    }
+  }
+
+  // Rendering the header and a list of tips
+
+  return (
+    <ScrollView contentContainerStyle={style.scrollViewContentContainer}>
+      <Text style={style.title}>Tips!</Text>
+      <TipsHeader schoolName={schoolName} />
+      <View style={style.section}>
+        {tipArray.map((tip) => (
```

Comment:
I think we have to pass in a key here to avoid this error

![image](https://github.com/DISC-NU/books-and-breakfast-app/assets/36573115/8f91b7ef-2404-4c6a-94f1-17e021a9016b)

It has something to do with how React renders

Reply:
https://stackoverflow.com/questions/55153873/warning-each-child-in-a-list-should-have-a-unique-key-prop